Package.swift

```diff
@@ -29,6 +29,7 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-log.git", from: "1.4.4"),
         .package(url: "https://github.com/apple/swift-atomics.git", from: "1.0.2"),
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
+        .package(url: "https://github.com/apple/swift-algorithms", from: "1.0.0"),
```

Comment:
I'm a bit on the fence about this dependency. Is it worth us considering not using it?

Reply:
We could also [copy & paste the implementation](https://github.com/apple/swift-algorithms/blob/fc8fdfd4dcc6d05f9d603c581fcbba643cb676df/Sources/Algorithms/Chunked.swift#L518-L824). Hasn't changed in ~2 years (except documentation).

Reply:
@FranzBusch thoughts on adding this dep?

Reply:
TBH, I think we should treat `swift-algorithms` similar to how we treat `swift-collections` or `swift-atomics`. They are stdlib extensions and follow the best practices around SemVer. IMO totally fine to pull this one in. In the end, any application will probably pull in `swift-collections`, `swift-atomics`, `swift-algorithms`, `swift-log`, `swift-metrics`, `swift-tracing` and in the future probably `swift-async-algorithms` and `swift-service-lifecycle`.